src/features/views/l10n/messageIds.ts

```diff
@@ -419,7 +419,14 @@ export default makeMessages('feat.views', {
       delete: m('Delete list'),
       editQuery: m('Edit Smart Search query'),
       makeDynamic: m('Convert to Smart Search list'),
-      makeStatic: m('Convert to static list'),
+      makeStatic: {
+        confirmDialogInfo: m(
+          'If you convert this list to a static list the Smart Search will be removed and the list will be empty, as all the people the Smart Search returned will no longer be there.'
```

Comment:
The part about the "empty" list is not 100% correct. If there are statically added rows from before, those rows will still be there. Try doing this:

1. Create a list
2. Add a person
3. Use ellipsis menu to convert to Smart Search list
4. Configure Smart Search and save
5. Use ellipsis menu again to convert to static list
6. Read the message
7. Confirm
8. Contemplate message vs result 😊